lib/plugin.zsh

```diff
@@ -104,3 +92,43 @@ geometry_plugin_render() {
 
   echo "$rprompt"
 }
+
+geometry_plugin_pin() {
+  if [[ $# -eq 0 ]]; then
+    echo "Error: Missing argument."
```

Comment:
Can you change this for something like "Missing the plugin name"? Just to be a tad bit more clear for when users experiment with this.